agent/engine/docker_container_engine.go

```diff
@@ -708,10 +708,13 @@ func (dg *dockerGoClient) ContainerEvents(ctx context.Context) (<-chan DockerCon
 				continue
 			case "restart":
 			case "resize":
-			case "destroy":
 			case "unpause":
 			// These result in us falling through to inspect the container, some
```

Comment:
This comment is now misplaced.  It should move below `case "destroy"` since it doesn't apply to that case.